documents/TestSuite/stdlib/channel/channels_attribute.mtlx

```diff
@@ -0,0 +1,58 @@
+<!--
+
+Tests the channel attribute.
+
+-->
+<?xml version="1.0"?>
+<materialx version="1.36">
+  <nodegraph name="float_to_color4_out" type="">
```

Comment:
For completeness can we add one test where the input is not a constant.
e.g. it's a stream or an image.

Reply:
Added 2 image based tests.